support-frontend/assets/pages/paper-subscription-checkout/helpers/__tests__/homeDeliveryDays.test.ts

```diff
@@ -0,0 +1,112 @@
+import { formatMachineDate } from 'helpers/utilities/dateConversions';
+import {
+	canDeliverOnNextDeliveryDay,
+	daysTillNextDelivery,
+	getHomeDeliveryDays,
+} from '../homeDeliveryDays';
+
+const DeliveryDays = {
+	Sunday: 0,
+	Monday: 1,
+	Tuesday: 2,
+	Wednesday: 3,
+	Thursday: 4,
+	Friday: 5,
+	Saturday: 6,
+};
+
+// 2019-02-25T06:22:32.198Z
+const monday = 1551075752198;
```

Comment:
Would it be worth changing this to `monday_2019_02_25` (for example) to make it clear later down what date your going FROM and TO?